umh-core/test/fsm/redpanda/state_transition_test.go

```
@@ -0,0 +1,513 @@
+						ClusterConfig: &redpanda_monitor.ClusterConfig{
+							Topic: redpanda_monitor.TopicConfig{
+								DefaultTopicRetentionMs:    1000000,
+								DefaultTopicRetentionBytes: 1000000000,
+							},
+						},
+						LastUpdatedAt: time.Now(),
+					},
+				},
+			}
+
+			tick = reconcileRedpandaUntilState(ctx, redpandaService, mockFileSystem, tick, s6fsm.OperationalStateRunning)
```

golangci-lint (1.64.8)

ineffectual assignment to tick

(ineffassign)